bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx

```diff
@@ -43,22 +87,17 @@ class VideoListItem extends Component {
             </DropdownTrigger>
             <DropdownContent placement="top left">
               <DropdownList className={styles.dropdownList}>
-                {[
-                  <DropdownListTitle className={styles.hiddenDesktop} key="name">
-                    {user.name}
-                  </DropdownListTitle>,
-                  <DropdownListSeparator className={styles.hiddenDesktop} key="sep" />,
-                  ...actions.map(action => (<DropdownListItem key={user.id} {...action} />)),
-                ]}
+                {availableActions}
               </DropdownList>
             </DropdownContent>
           </Dropdown>
           { user.isMuted ? <Icon className={styles.muted} iconName="unmute_filled" /> : null }
           { user.isListenOnly ? <Icon className={styles.voice} iconName="listen" /> : null }
         </div>
+        { showStats ? <VideoListItemStats toggleStats={this.toggleStats} getStats={getStats} stopGettingStats={stopGettingStats} videoTag={this.videoTag} /> : null }
```

Comment:
Since this is the component handling the display of the `VideoListItemStats` you probably should get the stats here and just pass down to `VideoListItemStats` instead of fetching them on  the `VideoListItemStats`.